packages/ag-trade/src/smartWallet.js

```diff
@@ -0,0 +1,394 @@
+/**
+ * @file Agoric smart wallet proxy
+ */
+// @ts-check
+
+import { E, Far } from '@endo/far';
+import {
+  SigningStargateClient,
+  assertIsDeliverTxSuccess,
+} from '@cosmjs/stargate';
+import { Tendermint34Client } from '@cosmjs/tendermint-rpc';
```

Comment:
This shouldn't have to be an `--UNSAFE` plugin -- the network stuff is all in a separate vat -- but I'm struggling to get protobuf stuff to run confined. See:

 - https://github.com/endojs/endo/discussions/1875